package.json

```diff
@@ -1,12 +1,8 @@
 {
-  "name": "gatsby-source-google-docs",
-  "version": "1.7.0",
+  "name": "@victsant/gatsby-source-google-docs",
```

Comment:
Please don't update the package.json name and version.
Use yarn instead of NPN. I don't want package-lock.json

Reply:
Sorry was testing something.